### ISSUE TITLE
Documentation - check help block example markup has correct aria-describedby 

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#813]
Any examples with mark up containing a help block should have the correct `aria-describedby` and `id` following the recent help block a11y changes.

**Comment by Stanton** [comment]
Checked, seems OK
